client/src/components/organizations/OrganizationSwitchPopover.vue

```diff
@@ -77,104 +136,145 @@ async function onOrganizationClick(org: ConnectedOrganization): Promise<void> {
     });
   }
 }
+
+async function onManageOrganizationClick(): Promise<void> {
```

Comment:
Why add this feature when it's already accessible right below the `OrganizationSwitchPopover` button ?

Reply:
I've tried to prevent all cases, so that if the user decides to instinctively click on the organization switch to access the main button, they can still access it. But I understand your question, maybe we can remove it for now.